lib/config/fileconf.go

```diff
@@ -2384,10 +2392,26 @@ func (o *OktaSync) Parse() (*servicecfg.OktaSyncSettings, error) {
 		return nil, trace.BadParameter("default owners must be set when access list import is enabled")
 	}
 
+	for _, filter := range o.GroupFilters {
+		_, err := utils.CompileExpression(filter)
+		if err != nil {
+			return nil, trace.Wrap(err, "error parsing group filter")
```

Comment:
for these compile errors, can we return the expression that caused the error in the message too?